modules/authentication/src/main/java/com/intuit/wasabi/authentication/impl/NoOpAuthenticateByHttpRequestImpl.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright 2017 Intuit
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package com.intuit.wasabi.authentication.impl;
+
+import com.intuit.wasabi.authentication.AuthenticateByHttpRequest;
+import com.sun.jersey.api.core.HttpRequestContext;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class NoOpAuthenticateByHttpRequestImpl implements AuthenticateByHttpRequest {
+
+    @Override
+    public Map<String, String> authenticate(HttpRequestContext httpRequestContext) {
+        return Collections.emptyMap();
```

Comment:
@abhishekjain88 Should we add a dummy unit test class for this to retain code coverage & overall testability? I know this is a nit. 😄